attestation-service/src/config.rs

```diff
@@ -23,6 +25,10 @@ pub struct Config {
     /// The Attestation Result Token Broker Config
     #[serde(default)]
     pub attestation_token_broker: AttestationTokenConfig,
+
+    /// Optional configuration for verifier modules
+    #[serde(default)]
+    pub verifier_config: Option<VerifierConfig>,
```

Comment:
Maybe we can move the verifier-specific config looking-up logic to the concrete verifier logic, like via env

https://github.com/confidential-containers/trustee/blob/main/deps/verifier/src/cca/mod.rs#L79-L80

And we can show this via README.md of nvidia verifier. This way can help us to avoid a mixing configuration file, part of whose fields are usually enabled by a feature flag.

Reply:
I think it's best to have all configuration in one place. It's more work for users if they have to set some environment variables and create new config files. I was thinking about changing CCA here, but it's out of scope of this PR.

I can add an example config to the docs.

Reply:
Good.